src/ice_shelf/MOM_ice_shelf.F90

```diff
@@ -2107,6 +2113,10 @@ subroutine initialize_ice_shelf(param_file, ocn_grid, Time, CS, diag, Time_init,
       'Heat conduction into ice shelf', 'W m-2', conversion=-US%QRZ_T_to_W_m2)
   CS%id_ustar_shelf = register_diag_field('ice_shelf_model', 'ustar_shelf', CS%diag%axesT1, CS%Time, &
       'Fric vel under shelf', 'm/s', conversion=US%Z_to_m*US%s_to_T)
+  CS%id_taux = register_diag_field('ice_shelf_model', 'taux_shelf', CS%diag%axesT1, CS%Time, &
+      'x-surface stress under the ice shelf', 'Pa', conversion=US%RLZ_T2_to_Pa)
```

Comment:
Because of the way these stresses are being calculated as the square root of the average of the squares, these diagnostics will always be non-negative, so perhaps the long description should change to "The magnitude of the x-surface stress under the ice shelf" or the diagnostic calculated in a way that preserves its sign.